hermes-consumers/build.gradle

```diff
@@ -10,12 +10,14 @@ dependencies {
     compile group: 'org.eclipse.jetty', name: 'jetty-client', version: '9.3.6.v20151106'
 
     compile group: 'javax.jms', name: 'javax.jms-api', version: '2.0'
-    compile group: 'com.jayway.jsonpath', name: 'json-path', version: '0.9.1'
+    compile group: 'com.jayway.jsonpath', name: 'json-path', version: '2.2.0'
     compile group: 'joda-time', name: 'joda-time', version: '2.5'
 
     compile group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.1'
     compile group: 'com.github.rholder', name: 'guava-retrying', version: '2.0.0'
 
+    compile group: 'com.wandoulabs.avro', name: 'wandou-avpath_2.11', version: '0.1.0'
```

Comment:
leaving dependency in consumer, jsonpth from hermes-test-helper was causing all sorts of trouble in all modules